src/components/Lesson/HorizontalLessonCard/types.ts

```diff
@@ -0,0 +1,6 @@
+import { LessonNormalized } from 'models/Lesson';
```

Comment:
This should be `Lesson`. Check Martagon's PR about this, as he had the same issue here: https://github.com/josgarmar28/z1-react-native/pull/5/files

Check the files and the new additions